MessageViewController/MessageView.swift

```diff
@@ -163,6 +168,13 @@ public final class MessageView: UIView, MessageTextViewListener {
         }
     }
 
+    public var maxScreenRatio: CGFloat = 0 {
+        didSet {
+            maxScreenRatio = 0...1 ~= maxScreenRatio ? maxScreenRatio : 0
```

Comment:
Wo! What is this syntax? Haha

Reply:
it checks if maxScreenRatio is within the range 0-1. If it isn't we default to 0 since we never want the message view frame to be bigger than our view frame.